Pipeline/PackageBuildOutputs/README.md

```diff
@@ -146,8 +152,108 @@ PackageBuildOutputs console output
+
+* `--generateWaziDeployAppManifest`
+* `--addExtension`
+* `--branch`
+* `--versionName` (recommended)
+
+```
++ groovyz /var/pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy \
+      --workDir /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs  \
+      --tarFileName retirementCalculator.tar  \
+      --application retirementCalculator  \
+      --addExtension \
+      --branch main \
+      --generateWaziDeployAppManifest  \
+      --includeLogs "*.log"  \
+      --versionName rel-2.0.0 \
+      --verbose
+```
+
+<details>
+  <summary>Console log</summary>
+
+PackageBuildOutputs console output
+
+```
+** PackageBuildOutputs start at 20240304.015055.050
+** Properties at startup:
+   addExtension -> true
+   application -> retirementCalculator
+   branch -> main
+   buildReportOrder -> [/u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/BuildReport.json]
+   copyModeMap -> ["COPYBOOK": "TEXT", "COPY": "TEXT", "DBRM": "BINARY", "LOAD": "LOAD", "JCL": "TEXT", "EQALANGX" : "BINARY"]
+   generateWaziDeployAppManifest -> true
+   includeLogs -> *.log
+   packagingPropertiesFile -> /u/pipeline/git/dbb/Pipeline/PackageBuildOutputs/packageBuildOutputs.properties
+   publish -> false
+   startTime -> 20240304.015055.050
+   tarFileName -> retirementCalculator.tar
+   verbose -> true
+   versionName -> rel-2.0.0
+   workDir -> /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs
+** Read build report data from /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/BuildReport.json.
+** Removing output records w/o deployType or with deployType=ZUNIT-TESTCASE
+** Deployable files detected in /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/BuildReport.json
+   ADO.RETIREME.MAIN.BLD.LOAD(EBUD01), LOAD
+*** Number of build outputs to package: 1
+** Copying build outputs to temporary package directory /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir
+   Copy ADO.RETIREME.MAIN.BLD.LOAD(EBUD01) to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/ADO.RETIREME.MAIN.BLD.LOAD/EBUD01.LOAD with DBB Copymode LOAD
+** Generate Wazi Deploy Application Manifest file to /u/ado/workspace/retirementCalculator/main/build-20240215.5/logs/tempPackageDir/waziDeployManifest.yaml
+---
+apiVersion: "wazideploy.ibm.com/v1"
+kind: "ManifestState"
+metadata:
+  name: "retirementCalculator"
+  description: "retirementCalculator"
+  version: "rel-2.0.0"
+  annotations:
+    creationTimestamp: "20240304.015055.050"
+    scmInfo:
+      type: "git"
+      uri: "git@ssh.dev.azure.com:v3/IBM-DAT/retirementCalculator/retirementCalculator"
+      branch: "main"
+      shortCommit: "cac60e95685575fab15a583d2a3966a087b58b38"
+    packageInfo: null
+artifacts:
+- name: "EBUD01"
+  description: "retirementCalculator/cobol/EBUD01.cbl"
+  properties:
```

Comment:
WD 3.0.2 will read the manifest in the tar file. The property path must be there and this property must give the relative path to the member in the tar file